src/pages/policy-details/policy-details.tsx

```diff
@@ -28,35 +32,73 @@ export default function PolicyDetails() {
   if (!policy) {
     return (
       <BaseLayout subtitle={`Policy ${policyId}`}>
-        <h4 className={styles.heading}>Policy {abbrStr(policyId)}</h4>
+        <h4 className={styles.heading}>Policy</h4>
         {status === 'loading' && <p className={globalStyles.secondaryColor}>Loading...</p>}
         {status === 'loaded' && <p>Unable to find your policy with given id.</p>}
       </BaseLayout>
     );
   }
 
+  const policyIpfsHref = `https://ipfs.io/ipfs/${policy.ipfsHash}`;
   return (
     <BaseLayout subtitle={`Policy ${policyId}`}>
-      <h4 className={styles.heading}>Policy {abbrStr(policy.policyId)}</h4>
+      <h4 className={styles.heading}>{policy.metadata}</h4>
       <BorderedBox
         noMobileBorders
         header={
           <Header>
             <h5>Details</h5>
-            <Button
-              size="large"
-              onClick={() => {
-                history.push(`/policies/${policy.policyId}/claims/new`);
-              }}
-            >
-              + Create a Claim
-            </Button>
+            {canCreateClaim(policy) ? (
+              <Button
+                variant="primary"
+                size="large"
+                onClick={() => history.push(`/policies/${policy.policyId}/claims/new`)}
+              >
+                + New Claim
+              </Button>
+            ) : (
+              <Tooltip overlay={<div>Claims are unable to be made for inactive policies.</div>}>
+                <div>
+                  <Button variant="primary" size="large" disabled>
+                    + New Claim
+                  </Button>
+                </div>
+              </Tooltip>
+            )}
           </Header>
         }
         content={
-          <div>
-            <p>TODO</p>
-            <pre>{JSON.stringify(policy, null, 2)}</pre>
+          <div className={styles.detailsSection}>
+            <div className={styles.detailsItem}>
+              <p className={globalStyles.bold}>Claimant Address</p>
+              <p className={globalStyles.secondaryColor}>{policy.claimant}</p>
+            </div>
+            <div className={styles.detailsItem}>
+              <p className={globalStyles.bold}>Beneficiary Address</p>
+              <p className={globalStyles.secondaryColor}>{policy.beneficiary}</p>
+            </div>
+            <div className={styles.detailsItem}>
+              <p className={globalStyles.bold}>Service Coverage Amount</p>
+              <p className={globalStyles.secondaryColor}>${formatUsd(policy.coverageAmountInUsd)}</p>
+            </div>
+            <div className={styles.detailsItem}>
+              <p className={globalStyles.bold}>Claims Allowed From</p>
+              <p className={globalStyles.secondaryColor}>{format(policy.claimsAllowedFrom, 'dd MMMM yyyy HH:mm')}</p>
```

Comment:
Might be worth extracting the time format we use to some constant, so we are consistent about it in the project, wdyt?

Reply:
I think that is overkill. You could create the function like this:
```ts

const DATE_FORMATS = {
BASIC: "dd MM yyyy",
RICH: "dd MM yyyy hh:mm:ss"
}

function formatDate(date: Date, format: keyof typeof DATE_FORMATS) {
  return baseFormat(date, format);
}
```

I consider these similar to CSS constants and having those at one place always proved to be worth it. But I don't think this codebase is at the point where doing this would be necessary.